src/anomalib/models/components/base/export_mixin.py

```diff
@@ -82,7 +84,8 @@ def to_torch(
             ... )
         """
         transform = transform or self.transform or self.configure_transforms()
-        inference_model = InferenceModel(model=self.model, transform=transform)
+        post_processor = post_processor or self.post_processor
+        inference_model = InferenceModel(model=self.model, transform=transform, post_processor=post_processor)
```

Comment:
If `post_processor` is going to be part of the lightning module, then do we need `InferenceModel`? Currently, `AnomalyModule` has a forward method that just calls `self.model(batch)`. I don't think we use this anywhere. And, `InferenceModel` has this as the `forward` method
```python
def forward(self, batch: torch.Tensor) -> torch.Tensor | tuple[torch.Tensor, torch.Tensor]:
        """Transform the input batch and pass it through the model."""
        batch = self.transform(batch)
        predictions = self.model(batch)
        return self.post_processor(predictions)
```

I feel this can just move here
https://github.com/openvinotoolkit/anomalib/blob/d1f824a5798262891dbbe583fb291e1cf9aa7d2a/src/anomalib/models/components/base/anomaly_module.py#L96